### PR TITLE
HOTFIX: minify css and js

### PR DESCRIPTION
**Pull request by xtine**
This forces minification of scss files (updating library) and uglification of js assets.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ function write(streams, name) {
   return concat(function(body) {
     return file(dest, body, {src: true})
       .pipe(rev())
-      .pipe(gulpif(production, uglify()))
+      .pipe(uglify())
```
Comment (noahmanger) [inline gulpfile.js]: I don't think we should remove the environment check here. Uglifying this locally makes it impossible to debug.

Comment (xtine) [inline gulpfile.js]: Hm, good point. I just checked and I think it's a case of the production variable not working correctly if prod isn't being uglified.

Comment (xtine) [inline gulpfile.js]: I believe FEC needs to be double checked because I think it's not being uglified on local for that as well.

Comment (noahmanger) [inline gulpfile.js]: Yeah seems like something is wrong. When I set that variable locally, it gets uglified. Can you remove it from this and we can deal with that issue separately?

**Comment by noahmanger** [comment]
Since we're doing a release this week, I think we should just merge into develop.

**Pull request merged**